features/environment.py

```diff
@@ -0,0 +1,18 @@
+from behave import *
+import ModelController
+
+def before_all(context):
+    context.MEPath = "/home/seanjkeene/PycharmProjects/doe-egot-me/"
```

Comment:
The context object is used by behave (the test automation system I'm using) and allows a convenient method of passing variables between different steps, features, or even files. This is used by steps.py, for example, so the user can configure their installation path. Using something more automated or a more global environmental variables file would be better, but this will do for now.